Toggl.Daneel/Views/SpiderOnARopeView.cs

```diff
@@ -255,20 +255,48 @@ private CGPath createCurvedPath(CGPoint anchor, CGPoint[] points)
             return path.CGPath;
         }
 
+        private Tuple<double, double> convertXYCoordinate(double originX, double originY,
```

Comment:
Why is the tuple here explicit?

```suggestion
        private (double, double) convertXYCoordinate(double originX, double originY,
```